phpstan-baseline.neon

```diff
@@ -181,102 +26,7 @@ parameters:
-			count: 1
-			path: tests/Contributors/Service/RepositoryTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$message of method PHPUnit\\\\Framework\\\\Assert\\:\\:assertTrue\\(\\) expects string, string\\|false given\\.$#"
-			count: 1
-			path: tests/Controller/Badge/DependentsControllerTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$message of method PHPUnit\\\\Framework\\\\Assert\\:\\:assertTrue\\(\\) expects string, string\\|false given\\.$#"
-			count: 1
-			path: tests/Controller/Badge/LicenseControllerTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$message of method PHPUnit\\\\Framework\\\\Assert\\:\\:assertTrue\\(\\) expects string, string\\|false given\\.$#"
-			count: 1
-			path: tests/Controller/Badge/SuggestersControllerTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$message of method PHPUnit\\\\Framework\\\\Assert\\:\\:assertTrue\\(\\) expects string, string\\|false given\\.$#"
-			count: 3
-			path: tests/Controller/Badge/VersionControllerTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$string of method PHPUnit\\\\Framework\\\\Assert\\:\\:assertRegExp\\(\\) expects string, string\\|null given\\.$#"
-			count: 3
-			path: tests/Controller/Badge/VersionControllerTest.php
-
-		-
-			message: "#^Method App\\\\Tests\\\\Controller\\\\HomeControllerTest\\:\\:provider\\(\\) return type has no value type specified in iterable type array\\.$#"
-			count: 1
-			path: tests/Controller/HomeControllerTest.php
-
-		-
-			message: "#^Parameter \\#1 \\$json of function json_decode expects string, string\\|false given\\.$#"
-			count: 1
-			path: tests/Controller/PackagistControllerTest.php
-
-		-
-			message: "#^Parameter \\#1 \\$json of function json_decode expects string, string\\|false given\\.$#"
-			count: 1
-			path: tests/Controller/SnippetControllerTest.php
-
-		-
-			message: "#^Property App\\\\Tests\\\\EventListener\\\\StatsSubscriberTest\\:\\:\\$persister has no typehint specified\\.$#"
-			count: 1
-			path: tests/EventListener/StatsSubscriberTest.php
-
-		-
-			message: "#^Property App\\\\Tests\\\\EventListener\\\\StatsSubscriberTest\\:\\:\\$controllerEvent has no typehint specified\\.$#"
-			count: 1
-			path: tests/EventListener/StatsSubscriberTest.php
-
-		-
-			message: "#^Property App\\\\Tests\\\\EventListener\\\\StatsSubscriberTest\\:\\:\\$request has no typehint specified\\.$#"
-			count: 1
-			path: tests/EventListener/StatsSubscriberTest.php
-
-		-
-			message: "#^Property App\\\\Tests\\\\EventListener\\\\StatsSubscriberTest\\:\\:\\$listener has no typehint specified\\.$#"
+			message: "#^Property Symfony\\\\Component\\\\HttpFoundation\\\\Request\\:\\:\\$headers \\(Symfony\\\\Component\\\\HttpFoundation\\\\HeaderBag\\) does not accept PHPUnit\\\\Framework\\\\MockObject\\\\MockObject&Symfony\\\\Component\\\\HttpFoundation\\\\ParameterBag\\.$#"
```

Comment:
This can be probably fixed using the generics `MockObject<ParameterBag>` definition. Not sure if it works with PHPUnit, it does with Prophecy + https://github.com/Jan0707/phpstan-prophecy/pull/165

Reply:
phpstan complains that "class PHPUnit\Framework\MockObject\MockObject is not generic"